### ISSUE TITLE
Accessibility test

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#556]

**Comment by brockfanning** [comment]
@SavvasStephanides will add a link to a feature branch for Phil to look at.

**Comment by SavvasStephanides** [comment]
Feature branch: http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/opensdg1/

Accessibility report: http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/opensdg1/accessibility/

@brockfanning Just noticed that the high contrast button disappeared. Any idea why?

**Comment by brockfanning** [comment]
I'm not sure the cause, but it looks like you've got a javascript error happening on the page.

**Comment by SavvasStephanides** [comment]
@brockfanning That's pretty weird since the errors seem to be from `open-sdg` code

**Comment by brockfanning** [comment]
It's most probably some interaction between the UK site and the latest Open SDG code. You might try removing these 2 files: https://github.com/ONSdigital/sdg-indicators/tree/opensdg1/_includes/assets/js/plugins

**Comment by SavvasStephanides** [comment]
Thanks for this @brockfanning. I'll have a look. In the meantime, the links above will do, or we could just use this link https://open-sdg.github.io/open-sdg-site-starter/

I've run an `A11yMachine` test on it: https://opensdg-starter-a11y-test.netlify.com/

**Comment by SavvasStephanides** [comment]
@brockfanning Hi Brock! I tried deleting those files and the contrast button came back! The graphs are still not working though and I'm getting this error in the console:

`Cannot read property 'attach' of undefined` 

http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/opensdg1/2-1-1/

**Comment by brockfanning** [comment]
@SavvasStephanides There is another javascript file being overridden, that you could try removing: https://github.com/ONSdigital/sdg-indicators/blob/opensdg1/_includes/assets/js/indicatorView.js

To be clear, when I recommend removing these files, I just mean as a quick fix to get this working. If you were actually upgrading to 1.0.0 for real, you'd want to think about why those files were overridden and decide whether you need to re-do any overrides based on the latest code.

**Comment by SavvasStephanides** [comment]
Thanks @brockfanning. That fixed it. We just need to get it working so we can test it for now. We'll discuss with the team on whether we need the overridden logic or not. Thanks!

**Comment by AnnCorp** [comment]
@SavvasStephanides  to move to sdg-indicators please

**Comment by brockfanning** [comment]
@SavvasStephanides Is it OK to close this?

**Comment by SavvasStephanides** [comment]
@brockfanning. Yes that's fine.

@AnnCorp [I added this](https://trello.com/c/UeyAWBkF) on our Trello board since that seems to be our main board for the UK site. Is this OK?

**Comment by AnnCorp** [comment]
Closing issue on open-sdg as will be UK specific task.
